server/parsec/components/postgresql/user_list_frozen_users.py

```diff
@@ -0,0 +1,67 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+from __future__ import annotations
+
+from parsec._parsec import OrganizationID, UserID, UserProfile
+from parsec.components.postgresql import AsyncpgConnection
+from parsec.components.postgresql.utils import Q, q_organization_internal_id
+from parsec.components.user import (
+    UserListFrozenUsersBadOutcome,
+)
+
+_q_get_profile_for_user = Q(
+    f"""
+SELECT
+    COALESCE(user_.current_profile, user_.initial_profile) AS profile,
+    user_.revoked_on
+FROM user_
+WHERE
+    organization = { q_organization_internal_id("$organization_id") }
+    AND user_id = $user_id
+"""
+)
+
+_q_get_organization_frozen_users = Q(
+    f"""
+SELECT
+    user_.user_id
+
+FROM user_
+WHERE
+    user_.organization = { q_organization_internal_id("$organization_id") }
```

Comment:
Not `auth_no_lock` returns a `AuthNoLockData` that contains the internal id of the organization, so you don't need to use  `q_organization_internal_id` \o/

Reply:
That's handy !